news/6141.feature

```diff
@@ -0,0 +1 @@
+Prefix user_log lines with their timestamp.
```

Comment:
Instead of "user_log," I would say something like "lines of the pip log file." I'm not sure user_log would be meaningful to many people as I think it's just the variable name in the code.

Reply:
It'll probably be wise to mention the `--log` option here as well, since that's probably the most user-recognizable part of this change.